packages/@lightningjs/ui-components/src/components/ProgressBar/ProgressBar.d.ts

```diff
@@ -18,23 +18,38 @@
 
 import lng from '@lightningjs/core';
 import Base from '../Base';
-import type { Color, StylePartial } from '../../types/lui';
+import { Color, StylePartial } from '../../types/lui';
 
-export type ProgressBarStyle = {
-  animationCurve: string;
-  animationDelay: number;
-  animationDuration: number;
+type ProgressBarStyle = {
+  animation: object;
   barColor: Color;
   progressColor: Color;
   radius: lng.Tools.CornerRadius;
 };
 
-export default class ProgressBar extends Base {
-  progress: string;
+declare namespace ProgressBar {
+  export interface TemplateSpec extends Base.TemplateSpec {
+    /**
+     * percentage of the current progress from 0 to 1
```

Comment:
I know this was copied over from the mdx but it doesn't quite make sense - we're passing a decimal value, not a percentage